pkg/resources/aws/aws.go

```diff
@@ -1450,10 +1509,183 @@ func DescribeELBs(cloud fi.Cloud) ([]*elb.LoadBalancerDescription, map[string][]
 	if innerError != nil {
 		return nil, nil, fmt.Errorf("error describing LoadBalancers: %v", innerError)
 	}
-
 	return elbs, elbTags, nil
 }
 
+// For NLBs and ALBs
+func ListELBV2s(cloud fi.Cloud, clusterName string) ([]*resources.Resource, error) {
+	elbv2s, _, err := DescribeELBV2s(cloud)
+	if err != nil {
+		return nil, err
+	}
+
+	var resourceTrackers []*resources.Resource
+	for _, elb := range elbv2s {
+		id := aws.StringValue(elb.LoadBalancerName)
+		resourceTracker := &resources.Resource{
+			Name:    id,
+			ID:      string(*elb.LoadBalancerArn),
+			Type:    TypeLoadBalancer,
+			Deleter: DeleteELBV2,
+			Dumper:  DumpELB,
+			Obj:     elb,
+		}
+
+		var blocks []string
+		for _, sg := range elb.SecurityGroups {
+			blocks = append(blocks, "security-group:"+aws.StringValue(sg))
+		}
+
+		blocks = append(blocks, "vpc:"+aws.StringValue(elb.VpcId))
+
+		resourceTracker.Blocks = blocks
+
+		resourceTrackers = append(resourceTrackers, resourceTracker)
+	}
+
+	return resourceTrackers, nil
+}
+
+func DescribeELBV2s(cloud fi.Cloud) ([]*elbv2.LoadBalancer, map[string][]*elbv2.Tag, error) {
+	c := cloud.(awsup.AWSCloud)
+	tags := c.Tags()
+
+	glog.V(2).Infof("Listing all NLBs and ALBs")
+
+	request := &elbv2.DescribeLoadBalancersInput{}
+	// ELB DescribeTags has a limit of 20 names, so we set the page size here to 20 also
```

Comment:
Is this comment also applicable to NLBs and ALBs? If so, please update the comment.